photon-server/src/main/java/org/photonvision/vision/processes/VisionModule.java

```diff
@@ -235,19 +234,18 @@ void saveAndBroadcastAll() {
         DataChangeService.getInstance()
                 .publishEvent(
                         new OutgoingUIEvent<>(
-                                UIUpdateType.BROADCAST,
-                                "fullsettings",
-                                ConfigManager.getInstance().getConfig().toHashMap(),
-                                null));
+                                "fullsettings", ConfigManager.getInstance().getConfig().toHashMap()));
     }
 
     void saveAndBroadcastSelective(WsContext originContext, String propertyName, Object value) {
         logger.trace("Broadcasting PSC mutation - " + propertyName + ": " + value);
+        if (originContext == null) {
+            logger.warn("Origin for selective send is null, broadcasting to all!");
```

Comment:
Warning seems a bit aggressive?

Reply:
was just for debugging. removing that now.